tools/importer/utils.js

```diff
@@ -64,11 +64,7 @@ export function addBreadCrumb(doc) {
   const breadcrumb = doc.querySelector('.section-breadcrumb');
 
   if (breadcrumb) {
-    // Not removing breadcrumb section from here because we need to extract breadcrumb title.
-    const cells = [['Breadcrumb']];
-    const table = WebImporter.DOMUtils.createTable(cells, doc);
-    breadcrumb.after(doc.createElement('hr'));
-    breadcrumb.replaceWith(table);
+    breadcrumb.remove();
```

Comment:
Should this function now be called `removeBreadCrumb` ?

Reply:
Yes